contracts/ronin/staking/BaseStaking.sol

```diff
@@ -33,25 +36,42 @@ abstract contract BaseStaking is
   uint256[49] private ______gap;
 
   modifier noEmptyValue() {
-    if (msg.value == 0) revert ErrZeroValue();
+    _requireValue();
     _;
   }
 
   modifier notPoolAdmin(PoolDetail storage _pool, address _delegator) {
-    if (_pool.admin == _delegator) revert ErrPoolAdminForbidden();
+    _notPoolAdmin(_pool, _delegator);
     _;
   }
 
   modifier onlyPoolAdmin(PoolDetail storage _pool, address _requester) {
-    if (_pool.admin != _requester) revert ErrOnlyPoolAdminAllowed();
+    _requirePoolAdmin(_pool, _requester);
     _;
   }
 
   modifier poolIsActive(address _poolAddr) {
```

Comment:
Please help me rename `poolIsActive` into `onlyActivePool` for adapting code convention.